applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/GrantApplications/Details.js

```diff
@@ -293,9 +293,16 @@ $(function () {
             assessmentResultWidgetManager.refresh();            
         }
     );
+    function initCustomFieldCurrencies() {
+        setTimeout(function () {
+            $('.custom-currency-input').maskMoney();
+        }, 1000);
+    }
+    
     PubSub.subscribe('application_assessment_results_saved',
         (msg, data) => {
-            assessmentResultWidgetManager.refresh();        
+            assessmentResultWidgetManager.refresh();
+            initCustomFieldCurrencies();
```

Comment:
there are other places where we refresh the assessmentResultWidgetManager.refresh but don't subsequently call the initCustomFieldCurrencies

Reply:
@AndreGAot - modified this code to use MutationObserver